README.md

```diff
@@ -1,30 +1,42 @@
 
-| Path                                                                   | Description                                                                                       |
-| ---------------------------------------------------------------------  | ------------------------------------------------------------------------------------------------- |
-| GET  /import-vat                                                       | Retrieve all import vat certificates                                                           |                
-| GET  /postponed-vat                                                    | Retrieve all postponed vat statements                                                          |                
-| GET  /adjustments                                                      | Retrieve all securities statements                                                           |                
+This service is built following GDS standards to [WCAG 2.2 AA](https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag)
 
+We use the [GOV.UK design system](https://design-system.service.gov.uk/) to ensure consistency and compliance through the project
 
 This applications lives in the "public" zone. It integrates with:
 
-* Secure Payments Service (SPS) / Enterprise Tax Management Platform (ETMP) via the [Customs Financials API](https://github.com/hmrc/customs-financials-api)
-* Secure Document Exchange Service (SDES) bulk data API via the [SDES proxy](https://github.com/hmrc/secure-data-exchange-proxy)
+Secure Payments Service (SPS) / Enterprise Tax Management Platform (ETMP) via the [Customs Financials API](https://github.com/hmrc/customs-financials-api)
+
+Secure Document Exchange Service (SDES) bulk data API via the [SDES proxy](https://github.com/hmrc/secure-data-exchange-proxy)
+
+## Running the service
+
+
+*From the root directory*
+
+`sbt run` - starts the service locally.
+`sbt runAllChecks` - Will run all checks required for a successful build
 
-In dev/test environments, the upstream services are stubbed out using stub services (see below).
+### Required dependencies
 
-## Running the application locally
+There are a number of dependencies required to run the service.
 
-The application has the following runtime dependencies:
+The easiest way to get started with these is via the service manager CLI - you can find the installation guide [here](https://docs.tax.service.gov.uk/mdtp-handbook/documentation/developer-set-up/set-up-service-manager.html)
+
+| Command                                          | Description |
+| --------                                         | ------- |
+| `sm2 --start CUSTOMS_FINANCIALS_ALL`             | Runs all dependencies |
+| `sm2 -s`                                         | Shows running services |
+| `sm2 --stop CUSTOMS_MANAGE_AUTHORITIES_FRONTEND` | Stop the micro service  |
```

Comment:
Better to use CUSTOMS_FINANCIALS_FRONTEND instead of CUSTOMS_MANAGE_AUTHORITIES_FRONTEND as better to mention the relevant service